routes/spentRoutes.js

```diff
@@ -0,0 +1,38 @@
+var db = require("../models");
+
+module.exports = function(app) {
+  app.get("/api/users/spent/:userid/", function(req, res) {
```

Comment:
@jasmine-fitts @clairestolp @tjordanwells I know we talked about this one previously, but after seeing the rest of the endpoints do you think we should just make this one "/api/spent/:userid"? Not sure if it's necessary, but figured I would ask before we merge.

Reply:
Yeah that does make sense to me. users is not necessary in the path.